doc/docfx.json

```diff
@@ -0,0 +1,77 @@
+{
+    "metadata": [
+        {
+            "src": [
+                {
+                    "files": [
+                        "source/scripting_v2/**.csproj"
+                    ],
+                    "exclude": [
+                        "**/bin/**",
+                        "**/obj/**",
+                        "_site/**"
+                      ],
+                      "src": "../"
+                }
+            ],
+            "dest": "scripting_v2"
+        },
+        {
+            "src": [
+                {
+                    "files": [
+                        "source/scripting_v3/**.csproj"
+                    ],
+                    "exclude": [
+                        "**/bin/**",
+                        "**/obj/**",
+                        "_site/**"
+                      ],
+                      "src": "../"
+                }
+            ],
+            "dest": "scripting_v3"
+        }
+    ],
+    "build": {
+        "content": [
+            {
+                "files": [
+                    "LICENSE.txt",
+                    "**.md",
+                    "**.yml",
+                    "scripting_v2/**.md",
+                    "scripting_v3/**.md",
+                    "scripting_v2/**.yml",
+                    "scripting_v3/**.yml",
+                    "scripting_v2/api/**.yml",
+                    "scripting_v3/api**.yml"
+                ]
+            }
+        ],
+        "dest": "_site",
+        "globalMetadataFiles": [],
+        "fileMetadataFiles": [],
+        "globalMetadata": {
+            "_appTitle": "ScriptHookVDotNet",
+            "__appFooter": "Copyright (C) 2015 crosire",
```

Comment:
`__appFooter`, `_gitContribute` and `_appLogoPath` appear to be unused?

Reply:
Should be used, well if I look now the footer isn't used. Also I put a line on the logo path as an workaround to remove the original logo since I haven't found a proper GTA V svg logo.

About the _gitContribuite I don't know yet how it really works so improve this doc link is broken. The view source for works just fine

Reply:
The footer had one `_` more and it didn't registered. `_gitContribute` was used, but since I've disabled contribuitors it won't be used anymore. Also if it happens to have a svg logo for the site, I'll add it ^^ Until then I'll use the `-` as a way to remove it